myenumerables.rb

```diff
@@ -0,0 +1,114 @@
+module Enumerable
+  def my_each
+    index = 0
+    while index < size
+      return to_a unless block_given?
+
+      yield self[index]
+      index += 1
+      self
+    end
+  end
+
+  def my_each_with_index(index = 0)
+    while index < size
```

Comment:
* [ ] There should be a working `my_each_with_index`  identical and returns the same thing as ruby's [`each_with_index`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-each_with_index). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each_with_index.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
- ```p [1,2,3,4,5,6].my_each_with_index``` should return same result as ```p [1,2,3,4,5,6].each_with_index```